OpenRA.Game/Server/Connection.cs

```diff
@@ -23,7 +23,7 @@ namespace OpenRA.Server
 {
 	public sealed class Connection : IDisposable
 	{
-		public const int MaxOrderLength = 131072;
+		public const int MaxOrderLength = 524288;
```

Comment:
Allowing 512kB seems a bit overkill. Perhaps add a comment explaining why it has to be so high and to then to reduce it once the problem is solved

Reply:
We could also expose this as an option for server hosts

Reply:
I think i'll just drop this change instead. The skirmish change is the important one here.

Reply:
Why wouldn't the multiplayer fix be important?

Reply:
But is it important enough to allow such a security compromising hack?